### PR TITLE
added tests server refactor and tests

### PR DESCRIPTION
**Pull request by minmaxdata**

### CODE REVIEW

File: __tests__/Filters.test.jsx [hunk 1]
```diff
@@ -1,9 +1,15 @@
 // should be able to sort by most recent
 // should be able to sort by number of stars
-import App from '../client/src/components/App';
+import Filters from '../client/src/components/Filters';
 
-describe('App', () => {
+const wrapper = shallow(<Filters />);
+
+describe('<Filters /> rendering', () => {
```
Comment (AndrewTjandra) [inline __tests__/Filters.test.jsx]: This test appears that it is checking that an 'h3' tag is being rendered in the Filter component vs the Filter component actually being rendered inside of App.  Might want to update test description.

File: __tests__/Reviews.test.jsx [hunk 2]
```diff
@@ -1,9 +1,19 @@
 // renders review title, text, ratings, reviewer name, date review is posted
 // should be have button to mark helpful
-import App from '../client/src/components/App';
+import Reviews from '../client/src/components/Reviews';
 
-describe('App', () => {
+const wrapper = shallow(<Reviews />);
+
+describe('<Reviews /> rendering', () => {
```
Comment (AndrewTjandra) [inline __tests__/Reviews.test.jsx]: Ditto to above comment, testing h3 is in review component vs Review being rendered in App.  Do you mean to test the component being rendered in App.  Maybe the test should be

const wrapper = shallow(`<App />`)
expect(wrapper.find(Reviews).toHaveLength(1)

File: server/index.js [hunk 3]
```diff
@@ -1,24 +1,34 @@
...
+
+const calculateRatings = (reviews) => {
+  const { ratings } = reviews;
+  console.log('ratings', ratings);
+  const { overall, quality, value, comfort, sizing, style } = ratings;
+  console.log('categories', overall, quality, value, comfort, sizing, style);
+  return ratings;
+};
 
 app.get('/', (req, res) => res.send('Hello World!'));
```
Comment (AndrewTjandra) [inline server/index.js]: Is this needed?  A GET request at the root should send the index.html file in your static folder.

File: __tests__/Filters.test.jsx (hunk 1 above)
Comment (minmaxdata) [reply __tests__/Filters.test.jsx]: yes, these tests aren't implemented yet. This is just a skeleton for further development.

File: __tests__/Reviews.test.jsx (hunk 2 above)
Comment (minmaxdata) [reply __tests__/Reviews.test.jsx]: yes, these tests aren't implemented yet. This is just a skeleton for further development.

File: server/index.js (hunk 3 above)
Comment (minmaxdata) [reply server/index.js]: this function isnt implemented yet - i was setting up server tests for its initial work.

File: __tests__/Reviews.test.jsx (hunk 2 above)
Comment (AndrewTjandra) [reply __tests__/Reviews.test.jsx]: awesome, thanks for the clarification.  Everything looks good =)

File: __tests__/Filters.test.jsx (hunk 1 above)
Comment (AndrewTjandra) [reply __tests__/Filters.test.jsx]: awesome, thanks for the clarification.  Everything looks good =)

**Comment by minmaxdata** [comment]
The tests arent implemented yet and the rating summary isnt built yet just working through them and getting the features worked out -

**Pull request merged**